cpu/stm32/periph/spi.c

```diff
@@ -256,8 +256,10 @@ static void _transfer_dma(spi_t bus, const void *out, void *in, size_t len)
     dma_wait(spi_config[bus].rx_dma);
     dma_wait(spi_config[bus].tx_dma);
 
-    /* No need to stop the DMA here, it is automatically disabled when the
-     * transfer is finished, only wait for SPI to leave the busy state */
+#ifdef DMA_CCR_EN 
```

Comment:
Travis is pointing out a trailing whitespace here.

Reply:
Should be ok now